Src/Particle/AMReX_ParticleContainerI.H

```diff
@@ -646,6 +646,36 @@ ParticleContainer<NStructReal, NStructInt, NArrayReal, NArrayInt, Allocator>::Re
   }
 }
 
+struct TransformerVirt
+{
+    /**
+       \brief This copies the particle but replaces the id with VirtualParticleID
+       *
+     **/
+    TransformerVirt ()
+    {}
+
+    template <typename DstData, typename SrcData>
+    AMREX_GPU_HOST_DEVICE
```

Comment:
AMREX_FORCE_INLINE

Reply:
Pushed a commit which adds the inline, and updates other inlines in Src/Particles for consistency